server/database/buildDummy.js

```diff
@@ -0,0 +1,48 @@
+const request = require("request");
+const fs = require("fs");
+const bcrypt = require("bcrypt");
+const userData = require("./buildData/teamsAndMatches");
+const dbConnection = require("./dbconnection");
+
+const makeEmptyTables = fs.readFileSync(`${__dirname}/build.sql`, "utf-8");
+
+const buildDatabase = () => {
+  dbConnection.query(makeEmptyTables, err => {
+    if (err) {
+      console.log("Building DB error", err);
+    } else {
+      gradData.forEach(user => {
+        bcrypt
```

Comment:
you could get rid of this code by hashing your passwords yourself and adding them to the json files, sounds like a lot of code but then you have less code to write

Reply:
I've completely deleted this file. I will hash them myself as you suggest. I will do that in the next pull request, if that's okay.